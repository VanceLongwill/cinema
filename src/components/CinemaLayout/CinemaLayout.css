.CinemaLayout {
  display: grid;
  grid-template-columns: repeat(5, minmax(50px, 1fr));
  grid-gap: var(--spacing-md);
}
