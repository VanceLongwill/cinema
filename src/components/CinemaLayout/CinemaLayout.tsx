import React from "react";

import "./CinemaLayout.css";

export const CinemaLayout: React.FunctionComponent = ({ children }) => {
  return <div className="CinemaLayout">{children}</div>;
};
