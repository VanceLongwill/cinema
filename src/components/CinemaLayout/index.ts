import { CinemaLayout } from "./CinemaLayout";
export default CinemaLayout;
