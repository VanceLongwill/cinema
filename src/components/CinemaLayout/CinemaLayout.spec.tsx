import * as React from "react";

import { mount } from "enzyme";

import { CinemaLayout } from "./CinemaLayout";

describe("CinemaLayout", () => {
  it("should render without errors", () => {
    expect(mount(<CinemaLayout />)).toMatchSnapshot();
  });
});
