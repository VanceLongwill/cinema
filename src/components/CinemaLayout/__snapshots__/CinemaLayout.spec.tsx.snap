// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CinemaLayout should render without errors 1`] = `
<CinemaLayout>
  <div
    className="CinemaLayout"
  />
</CinemaLayout>
`;
